Sat, 28 Dec 2019 09:55:56 - INFO - Connect to DB
Sat, 28 Dec 2019 09:55:56 - INFO - TRIZMA DB import
Sat, 28 Dec 2019 09:55:56 - INFO - Failed on TRIZMA DB import at: 2019-12-28_09-27-38, for following calling party: 4925137900221, for agent id: trz_sstankovic, with error:(pyodbc.DataError) ('22001', '[22001] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]String or binary data would be truncated. (8152) (SQLExecDirectW)')
[SQL: insert into records values('CC2Servicenummer','Belgrad','Nicht übermittelt','trz_sstankovic',                                  '4925137900221','','Nicht übermittelt','4925137900221_Saturday, December 28, 2019_10:55:53 AM')]
(Background on this error at: http://sqlalche.me/e/9h9h)
Sat, 28 Dec 2019 09:55:56 - INFO - 123TV DB import
Sat, 28 Dec 2019 10:00:34 - INFO - Connect to DB
Sat, 28 Dec 2019 10:00:34 - INFO - TRIZMA DB import
Sat, 28 Dec 2019 10:00:34 - INFO - 123TV DB import
Sat, 28 Dec 2019 10:12:28 - INFO - Connect to DB
Sat, 28 Dec 2019 10:12:29 - INFO - TRIZMA DB import
Sat, 28 Dec 2019 10:12:29 - INFO - 123TV DB import
